app/utils/cyaHelpers/transportMeans/TransportMeansCheckYourAnswersHelper.scala

```diff
@@ -95,10 +96,10 @@ class TransportMeansCheckYourAnswersHelper(
     id = Some("change-transport-means-inland-mode")
   )
 
-  def addDepartureTransportMeans: Option[SummaryListRow] = getAnswerAndBuildRow[Boolean](
-    page = AddDepartureTransportMeansYesNoPage,
+  def addDepartureTransportMeans(departureIndex: Index): Option[SummaryListRow] = getAnswerAndBuildRow[Boolean](
```

Comment:
This is a duplication of the method you've added in DepartureTransportMeansAnswersHelper? I don't think this page should have an index